firmware/raspberry_pi4_network_boot_beta.md

```diff
@@ -40,9 +40,11 @@ sudo echo FIRMWARE_RELEASE_STATUS="beta" > /etc/default/rpi-eeprom-update
 
 ### Configuration - Enable network boot
 Network boot is not enabled by default in the bootloader. To enable it the bootloader configuration file must be edited.
+
+**NOTE:** The `/lib/firmware...` line below ends with a date at the end of the file's name. We have a few versions of the beta bootloader in there, so make sure to select the most up-to-date version for your raspberry pi.
```

Comment:
Personally, I'd change "We have a few ..." to "There are several ..."

And "Raspberry Pi" ought to be capitalised :wink:

Reply:
Made those changes for you. Definitely makes more sense that way. Thanks!